games/2048.cpp/Portfile

```diff
@@ -0,0 +1,28 @@
+PortSystem              1.0
+
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+name                    2048.cpp
+version                 2019-12-12
+revision                0
+categories              games
+maintainers             {mails.ucas.ac.cn:chenguokai17 @chenguokai} openmaintainer
+
+homepage                https://github.com/plibither8/2048.cpp 
+platforms               darwin
+license                 MIT
+github.setup            plibither8 ${name} 81c0eebaa2e575c7c7943c01958b88896c9da852
```

Comment:
The `github.setup` needs to move to just above the current name line and it will then replace it. 

`github.setup            plibither8 2048.cpp 81c0eebaa2e575c7c7943c01958b88896c9da852
`

Reply:
It will also replace the `homepage` line which is set by `github.setup`.

Reply:
Moved, thanks